gremlin-test/features/branch/Union.feature

```diff
@@ -135,7 +135,6 @@ Feature: Step - union()
       | d[3].l   |
       | d[0].l   |
       | d[1.9].d |
-      | d[0].i   |
```

Comment:
just so i'm clear, this value is no longer in the output because vadas (aka `v[2]`) has no out edges to sum a weight on...is that right? if so, do you happen to know what catches the thrown `FastNoSuchElementException` and prevents it from bubbling up just kill the whole traversal? (i'm just trying to follow how the code works here for my own edification)

Reply:
> just so i'm clear, this value is no longer in the output because vadas (aka v[2]) has no out edges to sum a weight on...is that right?

Exactly.

It's the same thing as doing just `g.V(2).outE()` - the traverser would simply die as there are no outgoing edges. Likewise with `outE().values('weight').sum()`, there simply is no sum, hence the traverser dies.